scripts/getLatestDeadlineVersion.ts

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+const mod = require('../packages/aws-rfdk/lib/core/lambdas/nodejs/lib/version-provider');
+
+const provider=new mod.VersionProvider();
+
+provider.getVersionUris({ product: mod.Product.deadline, platform: mod.Platform.linux })
+  .then(result => {
+    const version = result.get(mod.Platform.linux);
+    console.log(`${version.MajorVersion}.${version.MinorVersion}.${version.ReleaseVersion}.${version.PatchVersion}`);
+  })
+  .catch(error => {
+    console.error(`ERROR - ${error.message}`);
```

Comment:
this will not set a non-zero exit code for the process since we are catching this error. Should we call [`process.exit(code)`](https://nodejs.org/api/process.html#process_process_exit_code) here so the caller has an easy way of identifying an error?